src/bootstrap/src/utils/dylib.rs

```diff
@@ -38,3 +38,16 @@ pub fn exe(name: &str, target: &str) -> String {
         name.to_string()
     }
 }
+
+/// Return the library name for the `run-make` support library for
+/// the given target.
+#[allow(dead_code)]
+pub fn run_make_support_lib_name(target: &str) -> &'static str {
+    if target.contains("windows") {
+        "librun_make_support.dll"
+    } else if target.contains("apple") {
+        "librun_make_support.dylib"
+    } else {
+        "librun_make_support.so"
+    }
+}
```

Comment:
I actually meant something like `fn exe` which accepts both a name and target. In this case the name that would be passed would be `"run_make_support"`.

Reply:
Ah sorry I misunderstood what you mean lol